src/Microsoft.TestPlatform.CoreUtilities/Helpers/EnvironmentHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CoreUtilities.Helpers
+{
+    using System;
+    using ObjectModel;
+
+    public class EnvironmentHelper
```

Comment:
there is an IEnvironment Interface in platform abstractions, should it not move there?

Reply:
We will go for PlatformAbstration if there is need to do #if defs. This function not required that.